scripts/dev

```diff
@@ -10,12 +10,6 @@ if [[ ! -d "${PWD}/config" ]]; then
     hass --config "${PWD}/config" --script ensure_config
 fi
 
```

Comment:
Why was this removed?

Reply:
This change is from PR #431. I've replaced usage of pythonpath with mounting custom_components in the devcontainer. Downside is that this only works inside the devcontainer.

It seems that setting adding integration to pythonpath will cause the integration package to shadow `nordpool` dependency, since pythonpath has precedence over site-packages. Integration will not load. 



`import nordpool` will point to the integration itself. I don't now if/why this was working in the earlier version of the devcontainer. Maybe some internal has changed in HASS, or if this relates to changes in python 3.12